core/views.py

```diff
@@ -45,69 +59,90 @@ def home(request):
     return render(request, "home.html", context)
 
 
-@ratelimit(key="user_or_ip", rate="1/m", block=True)
+# @ratelimit(key="user_or_ip", rate="1/m", block=True)
 def addHackathons(request):
+    print(request.method)
     if request.method == "POST":
         form = HackathonForm(request.POST)
+        print("got the post requets")
         if form.is_valid():
+            print("sending the form")
+            form.save(commit=False)
+            form.review_status = ReviewStatus.Pending
+            form.created_by = Hacker.objects.get(id=request.user.id)
+            form.save()
             return redirect("home")
+        else:
+            print(form.errors)
```

Comment:
add error handeling

Reply:
I am not quite sure how to do this, like I get the error so do I return it as a http response ?